saber/i2g/examples/I2G_Demo/jobPublic.yaml

```diff
@@ -16,7 +16,7 @@
 #------------------------------------------------
 # general:
 host_bossdb: api.bossdb.org
-token_bossdb: 9154f9b731940e65844ea3568a0d7daa94f06f8b
+token_bossdb: 9feb8eedea069626a7e96336d5dd8229a61cc99a
```

Comment:
is this the current public token? We should make sure this stays up to date on our wiki as well.

Reply:
I was using my personal token but I have now changed it to the public token!